src/control/cmd/daos/container.go

```diff
@@ -733,7 +733,7 @@ func printContainerInfo(out io.Writer, ci *containerInfo, verbose bool) error {
 	if verbose {
 		rows = append(rows, []txtfmt.TableRow{
 			{"Pool UUID": ci.PoolUUID.String()},
-			{"Container redundancy factor": fmt.Sprintf("%d", *ci.RedundancyFactor)},
+			{"Number of open handles": fmt.Sprintf("%d", *ci.NumHandles)},
```

Comment:
as i understand from this PR, you are changing the output of daos cont query to not include:
`Container redundancy factor : x`
and have instead
`Number of open handles   : x`
?

the RF is something useful really that users would like to see on cont query. im ok with the change to the info structure to remove that since we can grab that programmatically from the props, but visually on cont query it has been helpful to have the RF there..